proposals/lip-0018.md

```diff
@@ -26,29 +26,57 @@ Secondly, the addresses do not have a checksum or any other error detection mech
 
-Therefore, a new address system is proposed that eliminates the mentioned issues. The first and the third issue get resolved by using a 160-bit hash of the public key. The second one by using a checksum. Since the overall bit length of the hash value plus checksum is rather large, a base32 encoding is used. This results in shorter addresses than in hexadecimal representation while still being human-friendly.
+Therefore, a new address system is proposed that eliminates the mentioned issues. The first and the third issue get resolved by using a 160-bit hash of the public key. The second one by extending the address by a checksum on the user interface level. Since the overall bit length of the hash value plus checksum is rather large, a Base32 encoding is used on the user interface level. This results in shorter addresses than in hexadecimal representation while still being human-friendly.
 
 ## Specification
 
-### Address Computation
+The specification is split into three parts. The [first](#addresses-on-protocol-level) one defines an address format that is used on the protocol level. The [second](#address-encoding-for-user-interface-level) part defines an encoding of these address by adding a checksum and using a Base32 encoding that is proposed to used on the user interface level. The [third](#migration) part specifies the migration from the current address system to the new one.
 
-The address for an account, given by its public key, is generated as follows:
+### Addresses on Protocol Level
 
-1. Compute `SHA-256(pubkey),` where `pubkey` is the public key of the account, and truncate the output to its first 160 bits.
-2. Compute the checksum of the output of step 1 using the BCH code described below.
-3. Concatenate the output of step 1 and the output of step 2.
-4. Encode the output of step 3 into a specified base32 format (see below for details).
-5. Add the prefix "lsk" to the output of step 4.
+This section covers the definition of the address on the protocol level and further protocol related specifications. These addresses are also referred to as **binary addresses**.
 
-The output of step 5 is the address for the given account. An example of an address computation can be found in the [appendix](#address-computation). In the following subsections, some details of the steps above get clarified.
+#### Address Computation
+
+The binary address for an account with public key, `pubkey`, is generated by computing `SHA-256(pubkey)` and truncating the output to its first 160 bits.
+
+#### Usage in the Protocol
+
+The 160-bit binary addresses have to be used for the `recipientID` property of balance transactions. Moreover, if [LIP-0023](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0023.md) becomes active, then these addresses have to be used for 
```

Comment:
@AndreasKendziorra do we plan to change the convention `recipientID` to `recipientAddress`? if its which LIP is address them? if they are addressed can we update here to `recipientAddress`?

Reply:
Yes, it is planned to use this wording. It will be established with this one: https://research.lisk.io/t/define-schema-and-use-generic-serialization-for-transactions/211?u=andreas.kendziorra
I prefer to have it changed there because once this change it active changing a transaction property name is no protocol change anymore. Whereas in this LIP, it would imply another (not really needed) protocol change.